lib/widgets/atoms/mentor_card.dart

```diff
@@ -30,7 +30,9 @@ class MentorCard extends StatelessWidget {
             radius: Radii.avatarRadiusLarge,
             child: CircleAvatar(
               radius: Radii.avatarRadiusMedium,
-              backgroundImage: NetworkImage(avatarUrl.toString()),
+              backgroundImage: avatarUrl != null
+                  ? NetworkImage(avatarUrl!) as ImageProvider<Object>
```

Comment:
Do we need this casting here?

Reply:
Weirdly enough, yes, because of the ternary operator the type of the result cannot be determined... for some reason. 

`The argument type 'Object' can't be assigned to the parameter type 'ImageProvider<Object>?'`

Adding the cast to either result solves this.